[2025-04-16T15:46:53.433+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-04-16T15:46:54.289+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: beer_data_processing.check_file_exists scheduled__2025-04-16T04:00:00+00:00 [queued]>
[2025-04-16T15:46:54.570+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: beer_data_processing.check_file_exists scheduled__2025-04-16T04:00:00+00:00 [queued]>
[2025-04-16T15:46:54.574+0000] {taskinstance.py:2867} INFO - Starting attempt 5 of 6
[2025-04-16T15:46:54.767+0000] {taskinstance.py:2890} INFO - Executing <Task(FileSensor): check_file_exists> on 2025-04-16 04:00:00+00:00
[2025-04-16T15:46:55.678+0000] {logging_mixin.py:190} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70 DeprecationWarning: This process (pid=223) is multi-threaded, use of fork() may lead to deadlocks in the child.
[2025-04-16T15:46:55.806+0000] {standard_task_runner.py:72} INFO - Started process 620 to run task
[2025-04-16T15:46:55.642+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'beer_data_processing', 'check_file_exists', 'scheduled__2025-04-16T04:00:00+00:00', '--job-id', '606', '--raw', '--subdir', 'DAGS_FOLDER/beer_data_processing_dag.py', '--cfg-path', '/tmp/tmp2u_9kw6s']
[2025-04-16T15:46:55.997+0000] {standard_task_runner.py:105} INFO - Job 606: Subtask check_file_exists
[2025-04-16T15:46:59.094+0000] {task_command.py:467} INFO - Running <TaskInstance: beer_data_processing.check_file_exists scheduled__2025-04-16T04:00:00+00:00 [running]> on host 3e53738720c8
[2025-04-16T15:49:22.071+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='beer_data_processing' AIRFLOW_CTX_TASK_ID='check_file_exists' AIRFLOW_CTX_EXECUTION_DATE='2025-04-16T04:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='5' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-04-16T04:00:00+00:00'
[2025-04-16T15:49:22.352+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-04-16T15:49:22.370+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-04-16T15:49:22.391+0000] {logging_mixin.py:190} INFO - Current task name:check_file_exists state:running start_date:2025-04-16 15:46:54.294379+00:00
[2025-04-16T15:49:22.401+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing and current dag run status:running
[2025-04-16T15:49:22.424+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-04-16T15:49:22.476+0000] {baseoperator.py:423} WARNING - FileSensor.execute cannot be called outside TaskInstance!
[2025-04-16T15:49:23.321+0000] {base.py:84} INFO - Retrieving connection 'dataset_folder'
[2025-04-16T15:49:23.474+0000] {filesystem.py:109} INFO - Poking for file /opt/***/dags/data/beer_profile_and_ratings.csv
[2025-04-16T15:49:24.034+0000] {filesystem.py:113} INFO - Found File /opt/***/dags/data/beer_profile_and_ratings.csv last modified: 20250416145823
[2025-04-16T15:49:24.054+0000] {base.py:339} INFO - Success criteria met. Exiting.
[2025-04-16T15:49:24.064+0000] {filesystem.py:109} INFO - Poking for file /opt/***/dags/data/beer_profile_and_ratings.csv
[2025-04-16T15:49:24.349+0000] {filesystem.py:113} INFO - Found File /opt/***/dags/data/beer_profile_and_ratings.csv last modified: 20250416145823
[2025-04-16T15:49:25.983+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-04-16T15:49:26.079+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=beer_data_processing, task_id=check_file_exists, run_id=scheduled__2025-04-16T04:00:00+00:00, execution_date=20250416T040000, start_date=20250416T154654, end_date=20250416T154926
[2025-04-16T15:49:30.345+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-04-16T15:49:30.402+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-04-16T15:49:30.502+0000] {logging_mixin.py:190} INFO - Dag name:beer_data_processing queued_at:2025-04-16 15:07:06.513542+00:00
[2025-04-16T15:49:30.579+0000] {logging_mixin.py:190} INFO - Task hostname:3e53738720c8 operator:FileSensor
[2025-04-16T15:49:33.339+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-04-16T15:49:33.454+0000] {standard_task_runner.py:217} INFO - Process not found (most likely exited), stop collecting metrics
[2025-04-16T15:49:40.526+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-04-16T15:49:40.845+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
